finalPrep/annotations.js

```diff
@@ -2,41 +2,41 @@
+  var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;//create a variable named greeting and assign it the value of a string `Hey partner! My name is "" - will you be my friend?!` and interpolate that with the parameter 'name'
+  var demographics = [name, age];//create a variable named demographics and assign it a value of an array which includes the parameters 'name', and 'age'.
+  var powerSaying = "Did you know that I can " + specialPower + " ?";//create a variable named powerSaying and assign it a value of a string "Did you know that I can "" ?" and concatenate with the parameter 'specialPower'
+  var builtBear = {//create an object named builtBear
+    basicInfo: demographics,//create a key named basicInfo and assign it a value of the variable 'demographics'
+    clothes: clothes,//create a key named clothes and assign it a value of the parameter 'clothes'
+    exterior: fur,//create a key named exterior and assign it a value of the parameter 'fur'
+    cost: 49.99,//create a key named cost and assign it a 'number' value of 49.99
+    sayings: [greeting, powerSaying, "Goodnight my friend!"],//create a key named sayings and assign it a value of an array: the array should contain two variables 'greeting' and 'powerSaying', and a string "Goodnight my Friend!"
+    isCuddly: true,//create a key named isCuddly and assign it a boolean value of 'true'
   };
 
-  return builtBear
-}
+  return builtBear//create a return value for the function that will return, or log, the object named builtBear
+};
 
-buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
-buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');
+buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');//call the function buildABear and pass in the arguments ('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares')
+buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');//call the function buildABear and pass in the arguments ('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in')
 
 
 
 //FizzBuzz
-function fizzBuzz(num1, num2, range) {
-  for (var i = 0; i <= range; i++) {
-    if (i % num1 === 0 && i % num2 === 0) {
-      console.log('fizzbuzz');
-    } else if (i % num1 === 0) {
-      console.log('fizz');
-    } else if (i % num2 === 0) {
-      console.log('buzz');
-    } else {
-      console.log(i);
+function fizzBuzz(num1, num2, range) {//create a function named 'fizzBuzz' that takes 3 parameters: 'num1', 'num2', range
+  for (var i = 0; i <= range; i++) {//create a for loop with the inital expression set to 0, the condition to loop as long as less than or equal to range value, increment loop by 1
+    if (i % num1 === 0 && i % num2 === 0) {//create a conditional for the loop that evaluates if the modulo of i by num1 stricly equals 0 and  if the modulo of i by num2 also strictly equals 0
```

Comment:
Another way you could describe this condition  is "if the variable i is evenly divisible by num1 and num2"

Reply:
Solid work on technical language in your annotations but definitely something to keep improving since it's important in interviews.